submissions/AnnaStoyano/js-dom/script.js

```diff
@@ -0,0 +1,106 @@
+let download = function () {
+    let promise = fetch('./information.json');
+    promise.then(res => res.json())
+        .then(data => {
+            addMenuItems(data.faculties);
+            addContentItems(data.faculties);
+            return data;
+        })
+        .then(data=>{
+            menuHandler(data);
+            menuIconHandler();
+        })
+        
+}
+
+download();
```

Comment:
I'd recommend you using [DOMContentLoaded](https://developer.mozilla.org/uk/docs/Web/Events/DOMContentLoaded) event listener. You can place there your function `download`. 
Also, I'd recommend renaming this function, because to my mind, this name is too generic.